_init/index.js

```diff
@@ -66,7 +66,7 @@ const exec = __webpack_require__(871);
 
 async function run() {
     try {
-        const pwshFolder = __dirname.replace(/\/_init$/, '');
+        const pwshFolder = __dirname.replace(/_init$/, '');
```

Comment:
Did you test this change out on Linux?

Alternatively, couldn't you replace the slash with either a slash or backslash, such as:
```javascript
const pwshFolder = __dirname.replace(/[\\\/]_init$/, '');
```

Reply:
Yes it was tested with ubuntu also, but happy to make your suggested change.

Reply:
Actually not replacing the file path separator, is a bit cleaner if on the next line we don't add one before the action script:

```
const pwshFolder = __dirname.replace(/_init$/, '');
const pwshScript = `${pwshFolder}action.ps1`

```
or even do it on one line:

`const pwshScript = __dirname.replace(/_init$/, 'action.ps1');`

Reply:
Yep, I like approach the best!